code/modules/events/space_vines/vine_structure.dm

```diff
@@ -0,0 +1,196 @@
+// SPACE VINES (Note that this code is very similar to Biomass code)
+/obj/structure/spacevine
+	name = "space vine"
+	desc = "An extremely expansionistic species of vine."
+	icon = 'icons/effects/spacevines.dmi'
+	icon_state = "Light1"
+	anchored = TRUE
+	density = FALSE
+	layer = SPACEVINE_LAYER
+	plane = GAME_PLANE_UPPER_FOV_HIDDEN
+	mouse_opacity = MOUSE_OPACITY_OPAQUE //Clicking anywhere on the turf is good enough
+	pass_flags = PASSTABLE | PASSGRILLE
+	max_integrity = 50
+	var/energy = 0
+	/// Can this kudzu spread?
+	var/can_spread = TRUE
+	/// Can this kudzu buckle mobs in?
+	var/can_tangle = TRUE
+	var/datum/spacevine_controller/master = null
+	/// List of mutations for a specific vine
+	var/list/mutations = list()
+	var/trait_flags = 0
```

Comment:
```suggestion
	var/trait_flags = NONE
```

this is assuming these are bitflags, so use the define for zero here. also dmdoc